.github/workflows/release.yml

```diff
@@ -32,17 +32,40 @@ jobs:
             ~/.cargo/registry/cache/
           key: ${{ runner.os }}-cargo-build-stable-${{ hashFiles('**/Cargo.toml') }}
 
-      - uses: chainguard-dev/actions/setup-gitsign@main
+      - name: Install cargo-release
+        run: cargo install cargo-release
 
-      # Pulling rust toolchain & install linux dependencies
-      - uses: dtolnay/rust-toolchain@stable
```

Comment:
I think we might need this?

Reply:
i dont know. bevy doesnt have it so im curious